packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadButton.tsx

```diff
@@ -9,19 +9,21 @@ export interface MultipleFileUploadButtonProps extends React.HTMLProps<HTMLDivEl
   className?: string;
   /** Aria-label for the button */
   'aria-label'?: string;
+  'label'?: string;
```

Comment:
We'll need a description of the prop above the name. Maybe "Visible text label for the button"